app/uk/gov/hmrc/exports/services/mapping/ExportsPointerToWCOPointer.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.services.mapping
+
+import java.io.IOException
+import scala.io.Source
+
+object ExportsPointerToWCOPointer {
```

Comment:
This implementation is quite hard to read (lots of regexes and folds going on). Is this really more performant than the pattern match implementation?